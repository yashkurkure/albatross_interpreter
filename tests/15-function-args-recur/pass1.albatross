fun f3 void() {
  printstring("f3");
}

fun f2 void() {
  printstring("f2");
  f3();
}

fun f1 void () {
  printstring("f1");
  f2();
}

f1();
return 0;