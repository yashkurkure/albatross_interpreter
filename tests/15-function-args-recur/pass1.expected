f1f2f3
